Xamarin.Forms.Platform.Android/Renderers/ShellSectionRenderer.cs

```diff
@@ -130,6 +130,8 @@ public override AView OnCreateView(LayoutInflater inflater, ViewGroup container,
 			var shellSection = ShellSection;
 			if (shellSection == null)
 				return null;
+			if (shellSection.CurrentItem == null)
+				throw new ArgumentException("Active Shell Content not set. Have you added any Shell Content to your Shell?", nameof(ShellSection));
```

Comment:
The parameter to ArgumentException is the _parameter_ name. In this case it's a property name that isn't a parameter to this method at all. I think `InvalidOperationException` might be more suitable?

Reply:
@Eilon Yeah, `InvalidOperationException` makes sense here (didn't think about it). Should we also change the existing `ArgumentException` inside `OnCreateView()` method of `ShellItemRenderer.cs` (as `ShellItem` isn't a parameter there too)?

Reply:
Good question. It's a breaking change to change an exception type if it was already there. But I'm not sure what the policies are in this repo/project (I don't directly work on Xamarin.Forms). I'll let a resident Xamarin.Forms team member comment on what should be done here. (I personally find wrong exception types frustrating because it makes it hard to track down where the bug is. ArgumentException? What incorrect argument did I pass in?!? Oh, I didn't...)